src/smartHandler.ts

```diff
@@ -117,13 +117,15 @@ export class SMARTHandler implements Authorization {
                 `Authorization configuration not properly set up. Either 'tokenIntrospection' or 'jwksEndpoint' must be present`,
             );
         }
-
         const fhirUserClaim = get(decodedToken, this.config.fhirUserClaimPath);
         const patientContextClaim = get(decodedToken, `${this.config.launchContextPathPrefix}patient`);
+        const patientOrgsClaim = get(decodedToken, `patientOrgs`);
         const fhirServiceBaseUrl = request.fhirServiceBaseUrl ?? this.apiUrl;
 
         // get just the scopes that apply to this request
         const scopes = getScopes(decodedToken[this.config.scopeKey]);
+
+        // Remove scopes that do not have the required information to be useful or unused scopes
```

Comment:
Can we try to avoid making non-functional changes that could introduce conflicts? Thanks.